Fix grammar in storage-service-encryption.md (#125590)
Azure/azure-content: articles/storage/common/storage-service-encryption.md

# Azure Storage encryption for data at rest

> Azure Storage uses service-side encryption (SSE) to automatically encrypt your data when it is persisted to the cloud. Azure Storage encryption protects your data and helps you to meet your organizational security and compliance commitments.

Reviewer: ```suggestion
Azure Storage uses service-side encryption (SSE) to automatically encrypt your data when it's persisted to the cloud. Azure Storage encryption protects your data and helps you meet your organizational security and compliance commitments.
```
